lib/components/WizardProgress/WizardProgress.stories.tsx

```diff
@@ -15,6 +14,17 @@ export default meta;
 
 type Story = StoryObj<typeof meta>;
 
+const render = (args: Story['args']) => {
+  const [value, setValue] = useState<number>(args.currentStep);
+
+  useEffect(() => {
+    action('onChange')(args.currentStep);
```

Comment:
Hmm, this might not be wanted as there is no longer `onChange` prop for the component.

Reply:
Fixed